eng/performance/benchmark_jobs.yml

```diff
@@ -1,12 +1,14 @@
 parameters:
   osName: ''           # required -- windows | linux | macos
   osVersion: ''        # required -- OS version
-  kind: ''             # required -- benchmark kind. As of today, only "micro" benchmarks are supported, we plan to add "scenarios" soon
+  kind: ''             # required -- benchmark kind. As of today, only "micro" and "mlnet" benchmarks are supported, we plan to add "scenarios" soon
```

Comment:
You could always just default this to micro if you don't want to have to specify it for all of the legs, since most legs are micro.

Reply:
As for now yes, but we plan to add scenarios and other stuff so I would prefer to keep it as required with no default value.